### PR TITLE
Make sure the UI of the homepage matches the design and jump to behavior matches the one on slack

### PR DESCRIPTION
**Pull request by mayor04**
Changed the animation so it matches the design specified and also changed home page

https://user-images.githubusercontent.com/67462604/136688996-81ade916-e170-49c5-bb78-9022fe852b9f.mp4

### CODE REVIEW

File: lib/ui/nav_pages/home_page/home_page_viewmodel.dart [hunk 1]
```diff
@@ -249,7 +253,9 @@ class HomePageViewModel extends StreamViewModel {
   }
 
   void onJumpToScreen() {
-    navigationService.navigateTo(Routes.dmJumpToView);
+    navigation.navigateWithTransition(DmJumpToView(),
+        transition: NavigationTransition.DownToUp);
+    // navigationService.navigateTo(Routes.dmJumpToView);
```
Comment (jeremiahseun) [inline lib/ui/nav_pages/home_page/home_page_viewmodel.dart:258]: Please remove unused commented code

**Review by jeremiahseun — CHANGES_REQUESTED** [review]
Kindly show a video describing what you've done

File: lib/ui/nav_pages/home_page/home_page_viewmodel.dart (hunk 1 above)
Comment (mayor04) [reply lib/ui/nav_pages/home_page/home_page_viewmodel.dart:258]: done

**Comment by mayor04** [comment]
It's just a slight design change. I don't think video is needed for that

**Pull request merged**